client/js/screens/Profile/Profile.js

```diff
@@ -1,10 +1,39 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Image, Text, TouchableOpacity, ScrollView, View } from "react-native";
+import styles from "./styles";
+import { withNavigation } from "react-navigation";
 
-const Profile = ({ params }) => (
-  <View>
-    <Text>This is the Profile Screen</Text>
-  </View>
-);
+const Profile = ({ user, navigation }) => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.header}>
+        <Text style={styles.title}>My Profile</Text>
+        <TouchableOpacity
+          onPress={() => navigation.navigate("EditProfile", { user: user })}
+        >
+          <Text style={styles.link}>Edit</Text>
+        </TouchableOpacity>
+      </View>
+      <Image
+        borderRadius={75}
+        resizeMode="contain"
+        style={styles.profilePic}
+        source={require("../../assets/images/profile_pic.jpg")} // replace by profile pic once upload mutation is working
+      />
+      <View style={styles.info}>
+        <Text style={styles.userName}>
+          {user.firstName} {user.lastName}
+        </Text>
+        <Text style={styles.userLocation}>
+          {user.city}, {user.province}, {user.country}
+        </Text>
+        <Text style={styles.subheading}>My bike is a...</Text>
+        <Text style={styles.caption}>
+          {user.bikeColor} {user.bikeBrand} {user.bikeType}
+        </Text>
+      </View>
+    </View>
+  );
+};
 
-export default Profile;
+export default withNavigation(Profile);
```

Comment:
Pass the navigation prop from the screen container instead of using `withNavigation`. Thank you.

Reply:
Done.